sections/implementation/i18n.tex

```diff
@@ -0,0 +1,29 @@
+\section{Translations}
+To have a scalable application it is necessary to easily be able to translate the text to different languages.
+For this purpose we have used i18next, which is an internationalization framework for React and React native \cite{react-i18next}.
+Instead of hardcoding the text on every page, we use the component withTranslations and gets the function t.  
```

Comment:
Det er et internationalization for alle nodebaserede services, der er bare fx. en react extension til det